cmd/contour/servecontext.go

```diff
@@ -108,6 +108,14 @@ type serveContext struct {
 	// If the value is true, Contour will register for extensions/v1beta1.Ingress type and do
 	// the rewrite itself.
 	UseExtensionsV1beta1Ingress bool `yaml:"-"`
+
+	// Should Contour register to watch the new service-apis types?
+	// By default this value is false, meaning Contour will not do anything with any of the new
+	// types.
+	// If the value is true, Contour will register for all the service-apis types
+	// (GatewayClass, Gateway, HTTPRoute, TCPRoute, and any more as they are added)
+	// Marked EXPERIMENTAL as many places as possible.
+	UseServiceAPITypes bool `yaml:"-"`
```

Comment:
suggestion: if you call the flag `UseExperimentalServiceAPITypes` then you don't need the comments, its right there every time someone types the name of the flag.

Reply:
I didn't want to have to rename it all later, but that's a better idea,yeah.